packages/frontend/core/src/modules/workspace/properties/adapter.ts

```diff
@@ -36,9 +38,18 @@ export class WorkspacePropertiesAdapter {
     const rootDoc = workspace.docCollection.doc;
     this.properties = rootDoc.getMap(AFFINE_PROPERTIES_ID);
     this.proxy = createYProxy(this.properties);
+
+    this.properties$ = LiveData.from(
+      new Observable(observer => {
+        this.properties.observeDeep(() => {
+          requestAnimationFrame(() => observer.next(this.proxy));
```

Comment:
why use requestAnimationFrame, 

Performance optimization to avoid rendering should be done in react

Reply:
This may cause issue if this.proxy is a reference that never changes.

Reply:
Let me check. I added `requestAnimationFrame` because there is some UI view inconsistent issues. It seems like to be related to proxy here

Reply:
It seems that observeDeep is called in the same tick when doc data is changed. The change is somehow ignored and will not trigger react rerender. however requestAnimationFrame will get rid of the issue. will take some more time digging out why.

Reply:
I think you can keep it, I think we need to refactor this proxy in the future

Reply:
Oh I got it why it does not work - the proxy will firstly fire `observeDeep` and then set the value, which is the reason why in the callback it always gets the outdated data in the same tick: https://github.com/toeverything/blocksuite/blob/94ec11725132f96694c1ee83c3cfa64d697bb58d/packages/framework/store/src/reactive/proxy.ts#L157